### DISCUSSION TITLE
Help the Qortal guys incorporate Reticulum!

### DISCUSSION THREAD

**Discussion by Biggly** [markqvist/Reticulum#300] [General]
Qortal is a gutsy attempt at providing the world with a genuine alternative to our compromised Internet and domain names.

The coders say they could use mesh networking if you, Mark, would give them one hour of your time.

What will it take for you to do that?

Thanks.

**Comment by markqvist** [comment]
Relevant 32-post thread that you may be unaware of: https://github.com/markqvist/Reticulum/discussions/290

You *may* be interested in reading it.
